packages/dashboard/tsconfig.json

```diff
@@ -29,5 +29,10 @@
     "isolatedModules": true,
     "noEmit": true
   },
+  "ts-node": {
+    "compilerOptions": {
+      "module": "commonjs"
+    }
+  },
```

Comment:
Ha, I wound up having to do the same in my branch.

Reply:
I made this change per [this directive](https://webpack.js.org/configuration/configuration-languages/#typescript).